src/cmdfuncs.lua

```diff
@@ -137,6 +137,130 @@ function Avail(...)
    end
 end
 
+------------------------------------------------------------------------
+function Category(...)
+   dbg.start{"Category(", concatTbl({...},", "),")"}
+   local shell = _G.Shell
+
+
+   local cache = Cache:singleton{buildCache = true}
+   local moduleT, dbT = cache:build()
+
+   local categoryT = {}
+
+   local function add_module(cat, name)
+      if (cat == "") then return
+      elseif (categoryT[cat] == nil) then
+         dbg.print{"found new category: ", cat, "\n"}
+         categoryT[cat] = {}
+      end
+
+      if (not categoryT[cat][name]) then
+         dbg.print{"found new sn: ", name, " in category: ", cat, "\n"}
+         categoryT[cat][name] = 1
+      else
+         categoryT[cat][name] = categoryT[cat][name] + 1
+      end
+   end
+
+   for sn, v in pairs(dbT) do
+      for _, info in pairs(v) do
+         local category = info.Category or ""
+
+         for entry in category:split(",") do
+            entry = entry:trim()
+            add_module(entry, sn)
+         end
+      end
+   end
+
+   local masterTbl = masterTbl()
+   local search = ... and true
+   local twidth = TermWidth()
+   local cwidth = masterTbl.rt and LMOD_COLUMN_TABLE_WIDTH or twidth
+   local banner = Banner:singleton()
+
+   local a = {}
+
+   if (not search) then
+      dbg.print{"printing category block\n"}
+      a[#a+1] = "\n"
+      a[#a+1] = [[
+To get a list of every module in a category execute:
+   $ module category Foo
+      ]]
```

Comment:
```suggestion
where "Foo" is the name of a category.
      ]]
```
This is more like how it is done for `module keyword`